docs/DeveloperGuide.md

```diff
@@ -299,30 +299,36 @@ A successful execution of the `view` command is described as follows:
 
 #### Design Considerations
 ##### viewedPerson as FilteredList or a Person.
+**Aspect: Should viewedPerson be a FilteredList of a Person?**
 * **Alternative 1 (current choice):** viewedPerson as a FilteredList
     * Pros: Easier to implement, easier to develop if in the future we want to display more than one Person.
     * Cons: Not intuitive since now the viewSchedule command only support viewing one Person.
 * **Alternative 2:** viewedPerson as a Person
     * Pros: More intuitive because it **is** the displayed Person's Schedule we are interested in.
-    * Cons: Can only view one Person at any time, need to change the implementation when developer wants to
-  display more than one Person.
-##### Displayed Attributes upon calling viewSchedule
-* **Alternative 1 (current choice):** Display required attributes like Name, Phone, and Schedule; No Schedule on Person List
+    * Cons: Can only view one Person at any time, need to change the implementation when developer wants to display more than one Person.
+
+**Aspect: What attributes should be displayed in the right panel upon calling viewSchedule?**
```

Comment:
I think its a bit confusing to write right panel here but talk about Person List below because person list isn't in the right panel